SMU.cpp

```diff
@@ -338,6 +337,69 @@ int SessionItem::programmingModeDeviceExists(){
     return m_session->scan_samba_devs(samba_devs);
 }
 
+void SessionItem::usb_handle_thread_method(SessionItem *session_item)
+{
+    std::vector < Device* > last_devices = {};
```

Comment:
The constructor of a std::vector constructs an empty container by default. No need to initialize it with another empty one.